docs/reference/streamfield/blocks.rst

```diff
@@ -26,7 +27,7 @@ This document details the block types provided by Wagtail for use in :ref:`Strea
       ], block_counts={
           'heading': {'min_num': 1},
           'image': {'max_num': 5},
-      })
+      }, use_json_field=True)
```

Comment:
Should we update all `StreamField` examples to include `use_json_field`? I think just this one is enough, adding it to the others would add unnecessary noise...